cmd/stdiscosrv/apisrv.go

```diff
@@ -137,10 +137,10 @@ func (s *apiSrv) handler(w http.ResponseWriter, req *http.Request) {
 
 	if s.useHTTP {
 		// X-Forwarded-For can have multiple client IPs; split using the comma separator
-		forwardIPs := strings.Split(req.Header.Get("X-Forwarded-For"), ",")
-		if len(forwardIPs) > 0 {
-			// get the originating client (left most ip) and parse
-			remoteAddr.IP = net.ParseIP(forwardIPs[0])
+		forwardIP, _, found := strings.Cut(req.Header.Get("X-Forwarded-For"), ",")
+		if found {
```

Comment:
We should do this even if a comma is not found, no?

Reply:
You're right, we should.  Otherwise the code ignores the value when only one IP is provided.
Nice catch! 
I've pushed a commit for this